apis/r/DESCRIPTION

```diff
@@ -42,7 +42,7 @@ Imports:
     Matrix,
     stats,
     bit64,
-    tiledb (>= 0.20.3),
+    tiledb (>= 0.21.0),
```

Comment:
As that does not exist yet in repo, it would break CI etc.

Reply:
yup, ready and waiting for it to exist

Reply:
indeed! :)

Reply:
We could consider not imposing the constraint as it will break.

We can ensure in CI etc that 0.21.0 is present.  Fundamentally there is a difference between supplying newer version in the helper scripts such as `get_tarball.R` and imposing them to be present.